plugins/linode/personal_access_token.go

```diff
@@ -31,9 +33,34 @@ func PersonalAccessToken() schema.CredentialType {
 		DefaultProvisioner: provision.EnvVars(defaultEnvVarMapping),
 		Importer: importer.TryAll(
 			importer.TryEnvVarPair(defaultEnvVarMapping),
+			TryConfigFile(),
 		)}
 }
 
 var defaultEnvVarMapping = map[string]sdk.FieldName{
 	"LINODE_CLI_TOKEN": fieldname.Token,
 }
+
+// TryConfigFile looks for the token in the ~/.config/linode-cli file.
+func TryConfigFile() sdk.Importer {
+	return importer.TryFile("~/.config/linode-cli", func(ctx context.Context, contents importer.FileContents, in sdk.ImportInput, out *sdk.ImportAttempt) {
+		configFile, err := contents.ToINI()
+		if err != nil {
+			out.AddError(err)
+			return
+		}
+
+		fields := make(map[sdk.FieldName]string)
+		for _, section := range configFile.Sections() {
+			if section.HasKey("token") && section.Key("token").Value() != "" {
+				fields[fieldname.Token] = section.Key("token").Value()
```

Comment:
Looks like you don't need `fields` slice anymore as you populate candidate fields with a literal map (line 60).

Reply:
Gotten rid of that slicing.